i18n.js

```diff
@@ -0,0 +1,9 @@
+const NextI18Next = require("next-i18next").default;
+const path = require("path");
+
+module.exports = new NextI18Next({
+  otherLanguages: ["cy"],
+  localePath: path.resolve("./public/static/locales"),
+  keySeparator: false,
```

Comment:
What does this 'keySeparator' do?  I can't find an reference to it in the package repo

Reply:
I think it prevents full stops in the text from separating the keys

Reply:
Yeah that's right. The full stop is the default key separator and since we didn't need it as its mainly needed in nested jsons to separate keys, I just set it as false

Reply:
Great ok